### PR TITLE
Schema and music tests

### PR DESCRIPTION
**Pull request by Timowest12**
In this PR:

- Added tests for genre
- Added tests for music album
- Added tables to schema.sql
- Added foreign keys to tables

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -26,6 +26,42 @@ CREATE TABLE IF NOT EXISTS labels
...
+    genre_id INT,
+    source_id INT
+);
+
+
+-- Set _id in books table properties as foreign keys
+ALTER TABLE musicAlbum ADD CONSTRAINT fk_author_id FOREIGN KEY (author_id) REFERENCES authors(id);
+ALTER TABLE musicAlbum ADD CONSTRAINT fk_label_id FOREIGN KEY (label_id) REFERENCES labels(id);
+ALTER TABLE musicAlbum ADD CONSTRAINT fk_genre_id FOREIGN KEY (genre_id) REFERENCES genres(id);
+ALTER TABLE musicAlbum ADD CONSTRAINT fk_source_id FOREIGN KEY (source_id) REFERENCES sources(id);
```
Comment (Kingjosh007) [inline schema.sql:63]: The table name is musicalbum (all in lowercase).

Comment (Timowest12) [reply schema.sql:63]: OOh you're right, good work spotting that!

File: spec/genre_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require_relative '../classes/genre'
+
+describe Genre do
+  before :each do
+    @genre = Genre.new('country')
```
Comment (Kingjosh007) [inline spec/genre_spec.rb:5]: The constructor of Genre changed. I'll approve it and update it later. Just to let you know.

Comment (Timowest12) [reply spec/genre_spec.rb:5]: Thanks.

File: spec/musicalbum_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+require_relative '../classes/musicalbum'
+
+
+describe MusicAlbum do
+  before :each do
+    @musictest = MusicAlbum.new(2002-01-01,true,true,true)
```
Comment (Kingjosh007) [inline spec/musicalbum_spec.rb:6]: You should pass the date either as a String or as a Date object. You forgot the quotation marks around the date.

File: spec/musicalbum_spec.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  it 'music should be an instance of Music class' do
+    expect(@musictest).to be_an_instance_of MusicAlbum
+  end
+
+  it 'expect publish date to be correct' do
+    expect(@musictest.publish_date).to be(2002-01-01)
```
Comment (Kingjosh007) [inline spec/musicalbum_spec.rb:14]: You should pass the date either as a String or as a Date object. You forgot the quotation marks around the date.

**Review by Kingjosh007 — CHANGES_REQUESTED** [review]
Good job @Timowest12 ,
However, there are a few changes to be done before I approve it.

File: spec/musicalbum_spec.rb (hunk 3 above)
Comment (Timowest12) [reply spec/musicalbum_spec.rb]: This has been changed.

File: spec/musicalbum_spec.rb (hunk 4 above)
Comment (Timowest12) [reply spec/musicalbum_spec.rb]: This has been changed

**Comment by Timowest12** [comment]
Fixed all issues. 
Please let me know if there is anything else.

**Pull request merged**